### PR TITLE
Dockerize locust SIP log producer

### PR DESCRIPTION
**Pull request by keitaf**
- modify es_bulk to dynamically generate a bulk request body with SIP events
- add Dockerfile to build a docker image to run locust with es_bulk
- add sip_locust.yaml to deploy the locust docker image onto k8s cluster

tested locally and also on my EKS test cluster

### CODE REVIEW

**Comment by sergeyOkuntsev** [comment]
😍😍😍

File: sip_locust.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    metadata:
+      labels:
+        app: sip-locust
+    spec:
+      containers:
+        - name: locust
+          image: dashbase/locust:sip
+          env:
+            - name: LOCUST_OPTS
+              value: --no-web --clients 1 --hatch-rate 20 --run-time 1000h
```
Comment (sergeyOkuntsev) [inline sip_locust.yaml:21]: just to double check,  --hatch-rate is the nubmer of  http users/clients per second to spawn.  There is no reason to make it bigger than total --clients value .

File: es_bulk.py [hunk 2]
```diff
@@ -25,9 +24,7 @@ def __init__(self, parent):
     @locust.task(1)
     def es_bulk(self):
         try:
-            self.task_file = ESBULK.TASK_FILES.popleft()
-            with open(self.task_file) as f:
-                self.raw_data = ''.join(f.readlines())
+            self.raw_data = create_bulk_body(self.bulk_size)
```
Comment (sergeyOkuntsev) [inline es_bulk.py:27]: thanks for the suggestion, indeed, bulk real time generation is not affecting overall response time statistics.

File: sip_locust.yaml (hunk 1 above)
Comment (keitaf) [reply sip_locust.yaml:21]: oops i guess I put the values of `-c` and `-r` in opposite. :-P

**Pull request merged**